test/NinjaTools.FluentMockServer.API.Exclude.Tests/Downstream/RequestMapperSpec.cs

```diff
@@ -328,11 +303,11 @@ private void GivenInputRequestHasValidRequestUri([NotNull] string requestUri)
             _inputRequest.Scheme = uri.Scheme;
             GivenTheInputRequestHasHostWithPort(uri.Authority, uri.Port);
         }
-        
+
         private void GivenTheInputRequestHasBody([NotNull] Stream stream)
         {
             _inputRequest.Body = stream;
-        //    _inputRequest.EnableBuffering();
+            //    _inputRequest.EnableBuffering();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4739715)